curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/add-images-to-your-website.md

```diff
@@ -60,7 +60,7 @@ Your image element's `alt` attribute should not be empty.
 assert(
   $('img').attr('alt') &&
     $('img').attr('alt').length &&
-    /<img\S*alt=(['"])(?!\1|>)\S+\1\S*\/?>/.test(
+    /<[img|IMG]\S*alt=(['"])(?!\1|>)\S+\1\S*\/?>/.test(
```

Comment:
Square brackets make it match any character from `img|IMG`, incidentally this doesn't fail when answer is correct.

To match `img` or `IMG`, something like this is needed:
```suggestion
    /<(?:img|IMG)\S*alt=(['"])(?!\1|>)\S+\1\S*\/?>/.test(
```
It uses non-matching group `?:` to not change group number that's captured further in the pattern.

Reply:
I thought `<iMG>` or `<iMg>` are vaild tags mybad